### PR TITLE
Some CLI workflow formatting fixes

### PR DESCRIPTION
**Pull request by bergman**
# Hey, I just made a Pull Request!

## Description
Original output of cli was hard to parse with shell script because of `#` separator rather than standard ` ` (space).

I also found some of the contractions in the pretty workflow show output a bit cryptic so they were extended and also left-aligned (because I find it easier to read, but I guess that's subjective).

## Checklist for PR author(s)

- [x] Changes are covered by unit test
- [ ] All tests pass
- [x] Code coverage check passes
- [ ] Error handling is tested
- [ ] Errors are handled at the appropriate layer
- [ ] Errors that cannot be handled where they occur are propagated
- [ ] (optional) Changes are covered by system test
- [ ] Relevant documentation updated
- [ ] This PR has NO breaking change to public API
- [ ] This PR has breaking change to public API and it is documented

## Checklist for PR reviewer(s)

- [ ] This PR has been incorporated in release note for the coming version
- [ ] Risky changes introduced by this PR have been all considered

### CODE REVIEW

**Comment by bergman** [comment]
`WorkflowID Workflow#id` and `String Workflow#workflowId` is super confusing.

File: styx-cli/src/main/java/com/spotify/styx/cli/PlainCliOutput.java [hunk 1]
```diff
@@ -143,7 +143,7 @@ public void printWorkflow(Workflow wf, WorkflowState state) {
 
   @Override
   public void printWorkflows(List<Workflow> workflows) {
-    workflows.forEach(wf -> System.out.println(wf.id().toKey()));
+    workflows.forEach(wf -> System.out.println(wf.componentId() + " " + wf.workflowId()));
```
Comment (honnix) [inline styx-cli/src/main/java/com/spotify/styx/cli/PlainCliOutput.java]: This is a breaking change. Better do a code search to understand the impact.

Comment (bergman) [reply styx-cli/src/main/java/com/spotify/styx/cli/PlainCliOutput.java]: I couldn't find anyone that scripts the cli using the `--plain` or `-p` flag. A bunch are doing it without the flag though 😢
